### DISCUSSION TITLE
Wi-Fi is not working....

### DISCUSSION THREAD

**Discussion by ZETA-A** [sebanc/brunch#1309] [Q&A] [answered]
Hello!
I installed Chrome OS on the Bay Trail Atom tablet.
The version is r93, Bluetooth is working, but Wi-Fi is not working.
This is my screenshot below.
Thank you.
![IMG_0922](https://user-images.githubusercontent.com/56914556/141672546-27a0f6a8-bcdc-4326-9246-30df7b30036a.jpeg)
![IMG_0921](https://user-images.githubusercontent.com/56914556/141672547-7936db2f-a699-48ee-81be-8dc39b11e381.jpeg)
![IMG_0920](https://user-images.githubusercontent.com/56914556/141672548-f16b70f6-09cb-48f8-b741-5f81c1f3dbeb.jpeg)

**Comment by sebanc** [comment, marked as answer]
It has normally been fixed in the brunch-unstable branch:
https://github.com/sebanc/brunch-unstable/releases/tag/r94-unstable-20211114

**Reply by ZETA-A** [reply]
Thank you!
